src/main/java/com/google/sps/servlets/MentorEvidenceServlet.java

```diff
@@ -72,4 +82,36 @@ private void updateMentorEvidence(String paragraph) {
       logger.log(Level.SEVERE, exception.getMessage(), exception);
     }
   }
+
+  /**
+   * Adds a list of approvers (currently only the admins) to the mentor in the database.
+   */
+  private void addApprovers() {
+    int userId = Utility.getUserId();
+    // Create array to store IDs of approvers.
+    int[] approvers = {1, 2, 3};
+
+    for (int approverId : approvers) {
+      // Set up query to insert new experience tag to user.
+      String query = "INSERT INTO MentorApproval "
+          + "VALUES (?, ?, FALSE, FALSE)";
```

Comment:
you could declare the query outside since the loop does not affect its value

Reply:
also isn't the SQL syntax INSERT INTO Table(column1, column2...) ?

Reply:
It is! Done.